<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommandWidget</class>
 <widget class="QWidget" name="CommandWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>655</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::WheelFocus</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>4</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&amp;Name:</string>
       </property>
       <property name="buddy">
        <cstring>lineEditName</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditName">
       <property name="toolTip">
        <string>Command name shown in menu</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="IconSelectButton" name="buttonIcon"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="widgetCommandType" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Type:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButtonAutomatic">
        <property name="toolTip">
         <string>Run the command automatically if clipboard has new content</string>
        </property>
        <property name="text">
         <string extracomment="Type of command; triggered by whenever clipboard changes">Auto&amp;matic</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButtonInMenu">
        <property name="toolTip">
         <string>Show command in context menu of matching items</string>
        </property>
        <property name="text">
         <string extracomment="Type of command; triggered by a custom application shortcut">In M&amp;enu</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButtonGlobalShortcut">
        <property name="text">
         <string extracomment="Type of command; triggered by a custom global/system shortcut">Global Shortcut</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButtonIsScript">
        <property name="text">
         <string extracomment="Type of command; allows to extend scripting capabilities">Script</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButtonDisplay">
        <property name="text">
         <string extracomment="Type of command; allows change how items are displayed">Display</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widgetMenuShortcut" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="labelShortcut">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>&amp;Shortcut:</string>
        </property>
        <property name="buddy">
         <cstring>shortcutButton</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ShortcutButton" name="shortcutButton" native="true">
        <property name="focusPolicy">
         <enum>Qt::WheelFocus</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
     <zorder>shortcutButton</zorder>
     <zorder>labelShortcut</zorder>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widgetGlobalShortcut" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>&amp;Global Shortcut:</string>
        </property>
        <property name="buddy">
         <cstring>shortcutButtonGlobalShortcut</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ShortcutButton" name="shortcutButtonGlobalShortcut" native="true">
        <property name="focusPolicy">
         <enum>Qt::WheelFocus</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="tabBarAutoHide">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Comman&amp;d</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="CommandEdit" name="commandEdit" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy">
          <enum>Qt::WheelFocus</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabAdvanced">
      <attribute name="title">
       <string>&amp;Advanced</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>437</width>
            <height>412</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item alignment="Qt::AlignTop">
              <widget class="QGroupBox" name="groupBoxMatchItems">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>1</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="title">
                <string>Match Items</string>
               </property>
               <layout class="QFormLayout" name="formLayout_2">
                <property name="horizontalSpacing">
                 <number>2</number>
                </property>
                <property name="verticalSpacing">
                 <number>2</number>
                </property>
                <property name="leftMargin">
                 <number>4</number>
                </property>
                <property name="topMargin">
                 <number>4</number>
                </property>
                <property name="rightMargin">
                 <number>4</number>
                </property>
                <property name="bottomMargin">
                 <number>4</number>
                </property>
                <item row="0" column="0">
                 <widget class="QLabel" name="label_4">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>&amp;Content:</string>
                  </property>
                  <property name="buddy">
                   <cstring>lineEditMatch</cstring>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLineEdit" name="lineEditMatch">
                  <property name="toolTip">
                   <string>Skips the command if the input text does not match this regular expression (leave empty to match everything).

%2 through %9 (or argument[1] and up in script) in Command and Filter will be replaced with the captured texts.

Examples:

- Match URL: ^(https?|ftp)://
- Match PDF filenames: \.pdf$
- Match single character: ^.$
- Match remote multimedia: ^http://.*\.(ogv|vlc|mp4|mp3)$</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="labelWindow">
                  <property name="text">
                   <string>&amp;Window:</string>
                  </property>
                  <property name="buddy">
                   <cstring>lineEditWindow</cstring>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLineEdit" name="lineEditWindow">
                  <property name="toolTip">
                   <string>&lt;p&gt;Use command only for items copied to clipboard from window with title text that matches this regular expression (leave empty to match any window). On macOS, this contains the application name followed by a dash (&amp;quot;-&amp;quot;) then the window title. E.g. &amp;quot;Safari - GitHub&amp;quot;.&lt;/p&gt;</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_7">
                  <property name="text">
                   <string>For&amp;mat:</string>
                  </property>
                  <property name="buddy">
                   <cstring>comboBoxInputFormat</cstring>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QComboBox" name="comboBoxInputFormat">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="toolTip">
                   <string>Data of this MIME type will be sent to standard input of command.
Leave empty to disable this.</string>
                  </property>
                  <property name="editable">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="label_9">
                  <property name="text">
                   <string>&amp;Filter:</string>
                  </property>
                  <property name="buddy">
                   <cstring>lineEditMatchCmd</cstring>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="CommandEdit" name="lineEditMatchCmd" native="true">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>1</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="focusPolicy">
                   <enum>Qt::WheelFocus</enum>
                  </property>
                  <property name="toolTip">
                   <string>Skips the command if the filter command fails with non-zero exit code.</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBoxAction">
             <property name="title">
              <string>Action</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <property name="leftMargin">
               <number>4</number>
              </property>
              <property name="topMargin">
               <number>4</number>
              </property>
              <property name="rightMargin">
               <number>4</number>
              </property>
              <property name="bottomMargin">
               <number>4</number>
              </property>
              <item>
               <layout class="QFormLayout" name="formLayout_4">
                <property name="fieldGrowthPolicy">
                 <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
                </property>
                <item row="0" column="0">
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>Cop&amp;y to tab:</string>
                  </property>
                  <property name="buddy">
                   <cstring>comboBoxCopyToTab</cstring>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QComboBox" name="comboBoxCopyToTab">
                  <property name="toolTip">
                   <string>Name of tab to copy new items into (leave empty not to copy)</string>
                  </property>
                  <property name="editable">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBoxIgnore">
                <property name="toolTip">
                 <string>Remove matching item

Note: If this is applied automatically, no other automatic commands are executed.</string>
                </property>
                <property name="text">
                 <string>&amp;Remove Item</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBoxInMenu">
             <property name="title">
              <string>Menu Action</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <property name="spacing">
               <number>2</number>
              </property>
              <property name="leftMargin">
               <number>4</number>
              </property>
              <property name="topMargin">
               <number>4</number>
              </property>
              <property name="rightMargin">
               <number>4</number>
              </property>
              <property name="bottomMargin">
               <number>4</number>
              </property>
              <item>
               <widget class="QCheckBox" name="checkBoxHideWindow">
                <property name="toolTip">
                 <string>Hide window after command is activated from context menu of an item</string>
                </property>
                <property name="text">
                 <string>&amp;Hide main window after activation</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBoxCommandOptions">
             <property name="title">
              <string>Command options</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <property name="spacing">
               <number>2</number>
              </property>
              <property name="leftMargin">
               <number>4</number>
              </property>
              <property name="topMargin">
               <number>4</number>
              </property>
              <property name="rightMargin">
               <number>4</number>
              </property>
              <property name="bottomMargin">
               <number>4</number>
              </property>
              <item>
               <layout class="QFormLayout" name="formLayout">
                <property name="fieldGrowthPolicy">
                 <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
                </property>
                <property name="verticalSpacing">
                 <number>2</number>
                </property>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_8">
                  <property name="text">
                   <string>O&amp;utput:</string>
                  </property>
                  <property name="buddy">
                   <cstring>comboBoxOutputFormat</cstring>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QComboBox" name="comboBoxOutputFormat">
                  <property name="toolTip">
                   <string>Create items from standard output of the program (leave empty to disable)</string>
                  </property>
                  <property name="editable">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="separatorLabel">
                  <property name="text">
                   <string>&amp;Separator:</string>
                  </property>
                  <property name="buddy">
                   <cstring>lineEditSeparator</cstring>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QLineEdit" name="lineEditSeparator">
                  <property name="toolTip">
                   <string>Separator to match for splitting the output to multiple items</string>
                  </property>
                  <property name="text">
                   <string>\n</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="labelOutputTab">
                  <property name="text">
                   <string>Output &amp;tab:</string>
                  </property>
                  <property name="buddy">
                   <cstring>comboBoxOutputTab</cstring>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QComboBox" name="comboBoxOutputTab">
                  <property name="toolTip">
                   <string>Save items in tab with given name (leave empty to save in first tab)</string>
                  </property>
                  <property name="editable">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout">
                <property name="spacing">
                 <number>16</number>
                </property>
                <item>
                 <widget class="QCheckBox" name="checkBoxWait">
                  <property name="toolTip">
                   <string>Show action dialog before executing the command</string>
                  </property>
                  <property name="text">
                   <string>&amp;Wait</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="checkBoxTransform">
                  <property name="toolTip">
                   <string>Change item, don't create any new items</string>
                  </property>
                  <property name="text">
                   <string>Tr&amp;ansform</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelDescription">
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>134</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxShowAdvanced">
        <property name="text">
         <string>Show Advanced</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CommandEdit</class>
   <extends>QWidget</extends>
   <header>gui/commandedit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>IconSelectButton</class>
   <extends>QToolButton</extends>
   <header>gui/iconselectbutton.h</header>
  </customwidget>
  <customwidget>
   <class>ShortcutButton</class>
   <extends>QWidget</extends>
   <header>gui/shortcutbutton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEditName</tabstop>
  <tabstop>buttonIcon</tabstop>
  <tabstop>toolButtonAutomatic</tabstop>
  <tabstop>toolButtonInMenu</tabstop>
  <tabstop>toolButtonGlobalShortcut</tabstop>
  <tabstop>toolButtonIsScript</tabstop>
  <tabstop>toolButtonDisplay</tabstop>
  <tabstop>shortcutButton</tabstop>
  <tabstop>shortcutButtonGlobalShortcut</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>lineEditMatch</tabstop>
  <tabstop>lineEditWindow</tabstop>
  <tabstop>comboBoxInputFormat</tabstop>
  <tabstop>lineEditMatchCmd</tabstop>
  <tabstop>comboBoxCopyToTab</tabstop>
  <tabstop>checkBoxIgnore</tabstop>
  <tabstop>checkBoxHideWindow</tabstop>
  <tabstop>comboBoxOutputFormat</tabstop>
  <tabstop>lineEditSeparator</tabstop>
  <tabstop>comboBoxOutputTab</tabstop>
  <tabstop>checkBoxWait</tabstop>
  <tabstop>checkBoxTransform</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
